### ISSUE TITLE
An exception has occurred

### ISSUE THREAD

**Issue by estankus** [sf04/Binance-Trailing-Stop-Loss#4]
Hello

I am really new at using python
I added my keys to config.py file
I use "Spyder" to run the script main.py
I enter there needed argumets
and when I run main.py I get error: 
----------------------------------------------------------------------------------------
usage: main.py [-h] KEY/USDT 0.000550 sell 5
main.py: error: the following arguments are required: KEY/USDT, 0.000550, sell, 5
An exception has occurred, use %tb to see the full traceback.

SystemExit: 2
----------------------------------------------------------------------------------------

Should I use somthing else instead of "Spyder"? Or what could it be that I do wrong to get it to work?

Thanx for comment.

**Comment by orangebutblue** [comment]
I'm not familiar with Spydar, but it seems like you're not starting the script with the necessary arguments

**Comment by estankus** [comment]
What aplication do you prefer or would suggest to use to run the script?

**Comment by orangebutblue** [comment]
just to run the script? why not just run it directly from bash / cmd?

**Comment by estankus** [comment]
Just to be sure i am really doing all correct. I should run main.py from cmd?

Sent from Yahoo Mail on Android 
 
  On Sat, 27 Mar 2021 at 0:31, ***@***.***> wrote:   


just to run the script? why not just run it directly from bash / cmd?

**Comment by orangebutblue** [comment]
yes, just like the usage section of the Readme says

**Comment by estankus** [comment]
Hello,

thank you for support.
I try from cmd.
Looks like getting closer, but got error:
  File "C:\Users\kest\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.9_qbz5n2kfra8p0\LocalCache\local-packages\Python39\site-packages\ccxt\static_dependencies\ecdsa\ecdsa.py", line 56, in <module>
    from six import int2byte, b
ModuleNotFoundError: No module named 'six' 

what could be the cause, where could be the error?

**Comment by orangebutblue** [comment]
I'm guessing you don't know python at all. Then, why do you even want to use this project?
This is just a sample - meant to be used as part  of your trading bot. If you don't know any python, then this project will be useless to you.
Just use a finished trading bot instead

**Comment by estankus** [comment]
Yes, I am just a begginer, doing selftraining.I managed so far not so much, i prepared for myself auto buy/sell scripts when price, reaches target ptice, goes above/below selected MA or EMA value.To write script for trailing stop would be too much for me.I thought that your script is as a finished script for this.
Do you have any ready trading bot to suggest?






Sent from Yahoo Mail on Android 
 
  On Tue, 6 Apr 2021 at 21:09, ***@***.***> wrote:   


I'm guessing you don't know python at all. Then, why do you even want to use this project?
This is just a sample - meant to be used as part of your trading bot. If you don't know any python, then this project will be useless to you.
Just use a finished trading bot instead

**Comment by orangebutblue** [comment]
just look at FreqTrade. It's a completely finished and working bot. You can develop your own strategies in python.
Check out their discord, they have a good community.

**Comment by estankus** [comment]
Thank you for your attention and sugestion.Good luck!

Sent from Yahoo Mail on Android 
 
  On Tue, 6 Apr 2021 at 21:32, ***@***.***> wrote:   


just look at FreqTrade. It's a completely finished and working bot. You can develop your own strategies in python.
Check out their discord, they have a good community.
